Tools/CMake/AMReX_Config.cmake

```diff
@@ -147,19 +149,7 @@ function (configure_amrex AMREX_TARGET)
 
    # fast math
    if (AMReX_FASTMATH)
-       # GPU specific backends set in AMReXParallelBackends.cmake
-       if (AMReX_GPU_BACKEND STREQUAL NONE)
-           # See https://cmake.org/cmake/help/v4.1/variable/CMAKE_LANG_COMPILER_ID.html#variable:CMAKE_%3CLANG%3E_COMPILER_ID
-           target_compile_options(${AMREX_TARGET} PUBLIC
-               $<$<CXX_COMPILER_ID:AppleClang,Clang,CrayClang,GNU,IBMClang,IntelLLVM,XLClang>:-ffast-math>
-               $<${_cxx_msvc}:"/fp:fast">  # MSVC
```

Comment:
This also fixes a bug. The original code caused a compile error on MSVC:
```
       (ClCompile target) -> 
         \"\fp:fast"(1,1): error C1083: Cannot open source file: '"/fp:fast"': No such file or directory [D:\a\impactx\impactx\build\_deps\fetchedamrex-build\Src\amrex_3d.vcxproj]
```
https://github.com/BLAST-ImpactX/impactx/pull/1073